commands/debug/testdata/custom_hardware/arduino-test/samd/platform.txt

```diff
@@ -228,4 +228,4 @@ tools.gdb.path={runtime.tools.arm-none-eabi-gcc-7-2017q4.path}/bin/
 tools.gdb.cmd=arm-none-eabi-gdb
 tools.gdb.cmd.windows=arm-none-eabi-gdb.exe
 
-tools.gdb.debug.pattern="{path}/{cmd}" -ex 'target extended-remote | {tools.openocd.path}/{tools.openocd.cmd} -s "{tools.openocd.path}/share/openocd/scripts/" --file "{runtime.platform.path}/variants/{build.variant}/{build.openocdscript}" -c "gdb_port pipe" -c "telnet_port 0" -c init -c halt' {build.path}/{build.project_name}.elf
\ No newline at end of file
+tools.gdb.debug.pattern="{path}/{cmd}" --interpreter={interpreter}  -ex 'target extended-remote | {tools.openocd.path}/{tools.openocd.cmd} -s "{tools.openocd.path}/share/openocd/scripts/" --file "{runtime.platform.path}/variants/{build.variant}/{build.openocdscript}" -c "gdb_port pipe" -c "telnet_port 0" -c init -c halt' {build.path}/{build.project_name}.elf
```

Comment:
Let's add the default also inside platform.txt with:
```
tools.gdb.interpreter=console
```

Reply:
See https://github.com/arduino/ArduinoCore-samd/pull/506/commits/647cd2c2115a2d6cca306b126311023fcc55bf70